apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -638,17 +641,21 @@ async function processQuestionServer<T extends ExecutionData>(
     return { courseIssues, data, html: '', fileData: Buffer.from(''), renderedElementNames: [] };
   }
 
-  let result, output;
+  let result: any;
+  let output: string;
   try {
     ({ result, output } = await execPythonServer(codeCaller, phase, data, html, context));
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
👀 I'm trying to figure out what caused this change and similar ones. I backed it out and didn't get any type or lint errors.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/12620/commits/5671c6a44b2c579d0bce9c61b26863eea521035f

Reply:
This is all from the `course-db` stuff.

Reply:
Backed out in b6045cd.